notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>NIT: in the introduction section, knowing (no K)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>